packages/subproviders/test/integration/ledger_subprovider_test.ts

```diff
@@ -1,7 +1,7 @@
 import { JSONRPCResponsePayload } from '@0xproject/types';
 import { promisify } from '@0xproject/utils';
 import Eth from '@ledgerhq/hw-app-eth';
-// HACK: This depdency is optional and tslint skips optional depdencies
+// HACK: This dependency is optional and tslint skips optional dependencies
```

Comment:
Shouldn't we check that optional dependency if loaded? We can't just use it unconditionally

Reply:
We could check it and throw though the tests will fail if it isn't found anyway?

Reply:
Yeah, tests fail anyways with a helpful error message.